src/index.html

```diff
@@ -9,10 +9,68 @@
     <title>Product cards</title>
     <link
       rel="stylesheet"
-      href="./styles/index.scss"
+      href="./styles/style.scss"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
     />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div
+      class="card"
+      data-qa="card"
+    >
+      <div class="card__img">
+        <img
+          src="./images/imac.jpeg"
+          alt="iMac"
+        />
+      </div>
+
+      <div class="card__name">
+        <p>APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</p>
+      </div>
+
+      <div class="card__code">
+        <p>Product code: 195434</p>
+      </div>
+
+      <div class="card__review">
+        <div class="card__review-stars">
+          <div class="card__review-stars-star"></div>
+          <div class="card__review-stars-star"></div>
+          <div class="card__review-stars-star"></div>
+          <div class="card__review-stars-star"></div>
+          <div class="card__review-stars-star"></div>
+        </div>
```

Comment:
Need to use stars block from the previous task and add the modifier class
```suggestion
        <div class="stars stars--4">
          <div class="stars__star"></div>
          <div class="stars__star"></div>
          <div class="stars__star"></div>
          <div class="stars__star"></div>
          <div class="stars__star"></div>
        </div>
```